pom.xml

```
@@ -10,7 +10,7 @@
   </parent>
 
   <artifactId>cloudbees-folder</artifactId>
-  <version>${revision}${changelist}</version>
+  <version>6.${changelist}</version>
```

or
```suggestion
  <version>${changelist}</version>
```

CC @daniel-beck